test.sh

```diff
@@ -14,13 +14,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -e
+set -o errexit
+set -o xtrace
 
 # Linting
 ./buildifier.sh
 find . -name "*.py" | xargs pylint --disable=R,C
 
 # Bazel build and test
 bazel clean --host_force_python=PY2 --curses=no
+bazel build --host_force_python=PY2 --curses=no //package_manager:dpkg_parser.par
 bazel build --host_force_python=PY2 --curses=no //...
 bazel test --host_force_python=PY2 --curses=no --test_output=errors //...
```

Comment:
It feels weird to force to Python 2.x given it's EOL and may be removed from most systems.

Reply:
No idea about `--host_force_python=PY2`. If they are safe to remove, I'm willing to do so (in another PR).

Reply:
For those of us running on mac, --host_force_python=PY2 is necessary. Logs without
```
cwalker@mt100238 ~/Source/distroless (git)-[no-online-dpkg_parser] % bazel build --curses=no //package_manager:dpkg_parser.par
Loading:
Loading: 0 packages loaded
Analyzing: target //package_manager:dpkg_parser.par (1 packages loaded, 0 targets configured)
Analyzing: target //package_manager:dpkg_parser.par (28 packages loaded, 350 targets configured)
Analyzing: target //package_manager:dpkg_parser.par (28 packages loaded, 350 targets configured)
INFO: Analyzed target //package_manager:dpkg_parser.par (29 packages loaded, 445 targets configured).
INFO: Found 1 target...
[0 / 1] [Prepa] BazelWorkspaceStatusAction stable-status.txt
ERROR: /private/var/tmp/_bazel_cwalker/f470a75b3d336190f9075654c734a68a/external/subpar/compiler/BUILD:41:1: Building par file @subpar//compiler:compiler.par failed (Exit 1) compiler failed: error executing command bazel-out/host/bin/external/subpar/compiler/compiler --manifest_file bazel-out/host/bin/external/subpar/compiler/compiler.par_SOURCES --output_par ... (remaining 6 argument(s) skipped)

Use --sandbox_debug to see verbose messages from the sandbox
Traceback (most recent call last):
  File "bazel-out/host/bin/external/subpar/compiler/compiler", line 349, in <module>
    Main()
  File "bazel-out/host/bin/external/subpar/compiler/compiler", line 335, in Main
    ret_code = subprocess.call(args)
  File "/usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 172, in call
    return Popen(*popenargs, **kwargs).wait()
  File "/usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 394, in __init__
    errread, errwrite)
  File "/usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 1047, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory: '/usr/bin/python3'
Target //package_manager:dpkg_parser.par failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 5.953s, Critical Path: 0.38s
INFO: 0 processes.
FAILED: Build did NOT complete successfully
FAILED: Build did NOT complete successfully
```